s3_copy.sh

```diff
@@ -94,7 +94,7 @@ declare -a coreplugins=("org.ekstep.activitybrowser-1.3"
 
 for i in "${coreplugins[@]}"
 do
-   aws s3 --region ap-south-1 rm s3://ekstep-public-$1/content-plugins/$i/ --recursive
-   aws s3 --region ap-south-1 cp ansible/content-plugins/$i s3://ekstep-public-$1/content-plugins/$i --recursive --acl public-read
+   aws s3 --region $3 rm s3://$1/content-plugins/$i/ --recursive
```

Comment:
Hey define the $1, $2, $3 as variables and then use that variables inside the program
like
```
name=$1

echo $name
```